doc/manual/correctness.md

```diff
@@ -327,36 +377,42 @@ error: contract broken by a value.
 1 │ OptLevel
   │ -------- expected type
   │
-  ┌─ repl-input-4:3:27
+  ┌─ /path/to/config.ncl:4:26
   │
-3 │   opt_level | OptLevel = "0" ++ string.from_num level,
+3 │   opt_level | OptLevel = "O" ++ string.from_num level,
   │                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^ applied to this expression
   │
   ┌─ <unknown> (generated by evaluation):1:1
   │
-1 │ "04"
+1 │ "O4"
   │ ---- evaluated to this value
 ```
```

Comment:
Yeah, I assumed that the `0` was a deliberate mistake, but it was also never mentioned in the text, and an optimisation level of `O4` is still an error, as the contract only accepts up to `O3`, so I thought it might be simpler overall to focus only on catching that one error.

That said, using something like `A3` in the example might be an even more obvious mistake, so I'd be happy to change it to that.